smoke-tests/src/test/java/io/opentelemetry/smoketest/windows/WindowsTestContainerManager.java

```diff
@@ -0,0 +1,521 @@
+/*
```

Comment:
Now that you know how to do this, hopefully you can contribute it to test containers :D